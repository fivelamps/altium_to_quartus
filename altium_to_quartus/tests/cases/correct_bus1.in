NODENAME DATA.0$
    J2        72  U3       A1
NODENAME DATA.1$
    J2        70  U3       A2
NODENAME DATA.2$
    J2        72  U3       A3
NODENAME DATA.3$
    J2        70  U3       A4