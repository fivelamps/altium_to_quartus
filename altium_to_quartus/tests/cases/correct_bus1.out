set_location_assignment PIN_A1 -to "DATA[0]"
set_location_assignment PIN_A2 -to "DATA[1]"
set_location_assignment PIN_A3 -to "DATA[2]"
set_location_assignment PIN_A4 -to "DATA[3]"
